app/lib/common/widgets/html_editor/components/mention_item.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:acter/common/utils/constants.dart';
+import 'package:acter/common/widgets/html_editor/models/mention_type.dart';
+import 'package:acter_avatar/acter_avatar.dart';
+import 'package:flutter/material.dart';
+
+class MentionItem extends StatelessWidget {
+  const MentionItem({
+    super.key,
+    required this.mentionId,
+    required this.mentionType,
+    required this.displayName,
+    required this.avatarOptions,
+    required this.isSelected,
+    required this.onTap,
+  });
+
+  final String mentionId;
+  final MentionType mentionType;
+  final String displayName;
+  final AvatarOptions avatarOptions;
+  final bool isSelected;
+
+  final VoidCallback onTap;
+
+  @override
+  Widget build(BuildContext context) {
+    final isDesktop = desktopPlatforms.contains(Theme.of(context).platform);
+
+    return Container(
+      height: 60,
+      // selection color is only for desktop with keyboard navigation
+      color: (isSelected && isDesktop)
+          ? Theme.of(context).colorScheme.primary
+          : null,
```

Comment:
`isDesktop` is an incorrect check here: pads can also have keyboards and laptops might also have touch devices. what _I think_ you want to detect for here is a hardware keyboard, no?

but then also, how would `isSelected` even come about if not through a hardware-keyboard? I think this check is pointless and should be removed and the rest left to the outer part.


```suggestion
    return Container(
      height: 60,
      color: isSelected ? Theme.of(context).colorScheme.primary : null,
```

Reply:
I see, good point. I'll change the conditional 👍🏻 !